docs/book/src/user/quick-start.md

```diff
@@ -731,7 +731,7 @@ Calico is used here as an example.
 
 ```bash
 kubectl --kubeconfig=./capi-quickstart.kubeconfig \
-  apply -f https://docs.projectcalico.org/v3.20/manifests/calico.yaml
+  apply -f https://docs.projectcalico.org/v3.21/manifests/calico.yaml
```

Comment:
NOTE: I've verified the quickstart locally with Kubernetes 1.23 and Calico v3.21